### PR TITLE
Document $DOCKER_REGISTRY_URL

### PR DESCRIPTION
**Pull request by elgohr**

### CODE REVIEW

**Comment by elgohr** [comment]
Ping

File: login/README.md [hunk 1]
```diff
@@ -20,6 +20,18 @@ action "Docker Login" {
...
+```
+action "Docker Login" {
+  uses = "actions/docker/login@master"
+  secrets = ["DOCKER_USERNAME", "DOCKER_PASSWORD"]
+  env = {
+    DOCKER_REGISTRY_URL = "docker.pkg.github.com"
+  }
+}
+```
+
```
Comment (piki) [inline login/README.md]: This ☝️ added text seems to say the same as this 👇 that immediately follows it.  The proposed text might be a little clearer (the example is nice), but it's also redundant with what's below.  Did you mean to remove the "In addition..." paragraph?

Comment (elgohr) [reply login/README.md]: Sure, I was just trying to highlight the `DOCKER_REGISTRY_URL` a little more, as this is not possible in code examples.  
But I could also go without the text

Comment (piki) [reply login/README.md]: Let's fix the redundancy.  Can you remove both `A docker registry can be configured...` and `In addition, if you're logging into a registry...` and replace them with text above the example that says something like:

> In addition, if you're logging into a registry that's not Docker Hub, you can add `DOCKER_REGISTRY_URL` as either a secret or an environment variable to point to a different registry.

**Pull request merged**